collector/collector.go

```diff
@@ -44,6 +46,12 @@ type Config struct {
 	// How regularly the monitor should get root certificates from the Log.
 	// To disable getting roots, set to 0.
 	GetRootsPeriod time.Duration
+	// How regularly the monitor should submit a (pre-)certificate to the Log.
+	// To disable (pre-)certificate submission, set to 0.
+	AddChainPeriod time.Duration
+	// The CA to use to issue certificates for submission to the Log.  Only
```

Comment:
Suggested minor rewording: `The CA that issues certificates...`

Reply:
Then, `Must be set if AddChainPeriod != 0` as it must, right?

Reply:
Yup, done.